is_core/forms/widgets.py

```diff
@@ -375,3 +375,21 @@ def render(self, name, value, attrs=None):
     def value_from_datadict(self, data, files, name):
         value = super(MultipleTextInput, self).value_from_datadict(data, files, name)
         return [v.strip() for v in value.split(self.separator)] if isinstance(value, six.string_types) else value
+
+
+class AbstractDateRangeWidget(MultiWidget):
+
+    def __init__(self, filter_name):
+        class_from, class_to = self.get_range_classes()
+        super(AbstractDateRangeWidget, self).__init__(
+            (
+                TextInput(attrs={'class': class_from, 'data-filter': '{}__gte'.format(filter_name)}),
```

Comment:
mělo by stačit udělat DateRangeFromWidget a DateRangeToWidget a v metodě build_attrs napsat kousilíneček kódu

```
def build_attrs(self, **kwargs):
   attrs = super().build_attrs(**kwargs)
   attrs['data-filter'] = '{}__{}'.format(attrs['data-filter'], operator_string)
```

kde operator_string je gte nebo lte